README.md

```diff
@@ -59,4 +59,4 @@ If you find worth it then please star and share it.
 
 ## Your Contribution Most Welcome
 
-Please send PR if you find some important but please at least once see the [contribute guide](https://github.com/jsroyal/EngineeringBlog/blob/master/CONTRIBUTING.md).
```

Comment:
Why is it showing update?

Reply:
Not sure... I think it was removed and added back?

Reply:
Ok !!